### PR TITLE
Homepage improvements

### PR DESCRIPTION
**Pull request by seankross**
## Description
@tefirman @sckott @sitapriyamoorthi I added more prose to the home page but now I feel like I don't like how the homepage looks, what do ya'll think? Suggestions extremely welcome.

## Related Issue
Working on #9.

### CODE REVIEW

**Comment by sckott** [comment]
>  I don't like how the homepage looks

Can you be more specific? 

Some ideas:
- break up the prose into sections. for example, **Our Mission** and **Our Contribution** could be broken into sections of the page more distinctly by using color/lines/etc. e.g., here https://ucsc-ospo.github.io/ 
- possibly we should have a slightly bigger font size for the body text? seems kinda small

**Comment by vortexing** [comment]
See issues #15  and #14 for some thoughts too!

**Comment by tefirman** [comment]
Couple ideas, but obviously feel free to take or leave any of them:

- I'm not seeing the WILDS email address anywhere, should we add that to the "Help" and/or "Get Involved" sections?
- Might not be worth the effort of switching templates, but I wonder if the navigation bar along the top would look better as a sidebar, similar to how the contribution guide is laid out.

**Comment by sckott** [comment]
> I wonder if the navigation bar along the top would look better as a sidebar,

I think that would make it look like a book which is okay, but I think we're going for more of a website look, yeah?

**Comment by seankross** [comment]
I'm much happier now with this content and layout for the homepage.

File: index.qmd [hunk 1]
```diff
@@ -20,21 +20,27 @@ anim_width <- function(x, width1, width2) {
...
+**S**oftware, which highlights our focus on people in their day-to-day 
+scientific work. The mission of WILDS includes the following:
 
-- Engage with people developing open-source software across the Cancer
-Consortium.
+- Engage with the Cancer Consortium community about how they can adopt open
+science practices into their own workflows.
+- Educate our community about the critical role that open source software plays
+in biomedical data science practice.
+- Support open-source software developers across the Cancer Consortium.
```
Comment (tefirman) [inline index.qmd:43]:
```suggestion
- Support open source software developers across the Cancer Consortium.
```
Looks like we're switching from "open-source" to "open source", just want to make sure we catch all of the stragglers, I think there are a couple more below this.

File: index.qmd [hunk 2]
```diff
@@ -43,11 +49,32 @@ academically publish, and publicize their software.
...
+
+To support our mission, WILDS is actively engaged in the following activities:
+
+- We maintain, distribute, and publish open source scientific software, mostly
+implemented in the Python and R ecosystems.
+- We maintain and distribute open source scientific workflows implemented in
+WDL or Nextflow. 
+- We develop [research compendia][rescomp] to demonstrate common data analyses
+and how they can best be organized and shared.
+- We package and develop Docker recipies and Dockerfiles so that researchers
```
Comment (tefirman) [inline index.qmd:62]:
```suggestion
- We package and develop Docker recipes and Dockerfiles so that researchers
```

**Review by tefirman — APPROVED** [review]
Looks awesome, totally agree that this new layout looks much better! Just added a few very minor comments, feel free to take or leave them. Thanks @seankross !

**Pull request merged**